### ISSUE TITLE
Copy contrast ratio from UI

### ISSUE THREAD

**Issue by OLLI-S** [ThePacielloGroup/CCAe#168]
**Is your feature request related to a problem? Please describe.**
I would like to copy the contrast ratio (just the contrast ratio, like 8.6:1) to the clipboard.

**Describe the solution you'd like**
In my reports I have to insert the contrast ratio for many elements, so it would be easier if I can copy the  contrast ratio (just the  contrast ratio) with one click into the clipboard.
The field "Contrast Ratio" should be an read-only input field without borders, so I can mark the contrast ratio and copy it to the clipboard.
You can see this feature in many other applications, like Mozilla Firefox:

![grafik](https://user-images.githubusercontent.com/8037790/82412050-33d90a80-9a73-11ea-88ae-f9c5e27ab30b.png)

**Describe alternatives you've considered**
Maybe you add a button so I copy the  contrast ratio with one click.
The **easiest solution** would be the hot key **Ctrl+C** to copy the  contrast ratio (without any other information) into the clipboard.

**Comment by ferllings** [comment]
Yes, currently you can use ctrl(or cmd)+shift+c to copy the full result view.
```
Foreground: rgba(0, 0, 0, 1)
Background: #FFFFFF
The contrast ratio is: 21:1
1.4.3 Contrast (Minimum) (AA)
    Pass for large and regular text
1.4.6 Contrast (Enhanced) (AAA)
    Pass for large and regular text
1.4.11 Non-text Contrast (AA)
    Pass for UI components and graphical objects
```
I was planning to make this output customizable (like for the old legacy Mac version)

**Comment by OLLI-S** [comment]
@ferllings I know the ctrl+shift+c to copy the results including additional information.
But here I just need the contrast ratio (8.6:1).

**Comment by ferllings** [comment]
Yes, that why I said I want to make it customizable, so you can what information you want to copy/past.

**Comment by OLLI-S** [comment]
I suggest that you just **add the new shortcut Ctrl+C** so you have two shortcuts:
* **Ctrl+C** to copy the contrast ratio (**without** any other information)
* **Ctrl+Shift+C** to copy the results **including** additional information

This is easier for the user (he does not have to configure anything and has both options available).

**Comment by ferllings** [comment]
The issue with using Ctrl+C, is that you can't copy any other information on the UI, like the color values, the WCAG description text, etc...

**Comment by OLLI-S** [comment]
It is also OK if you make it configurable.
So I would set up to copy just the color contrast and other users can use the default option (copy everything).

**Comment by ferllings** [comment]
Fixed with the customizable copy results 37fc9a68cc11d010012ce695e6fdd9b365127e25
